spec/secret_code_spec.rb

```diff
@@ -23,58 +25,50 @@
   # not sure if i should be testing this?
   describe '#count_pins' do
     context 'will correctly count up the pins' do
-      xit 'when all pins white' do
+      it 'when all pins white' do
         feedback = Array.new(4) {'red'}
-        count_pins(feedback)
-        # expect(red_pin).to_eq 4 && white_pin.to_eq 0
+        expect(count_pins(feedback)).to_eq [4, 0]
       end
-      xit 'when all pins red' do
+      it 'when all pins red' do
         feedback = Array.new(4) {'white'}
         count_pins(feedback)
-        # expect(red_pin).to_eq 4 && white_pin.to_eq 0
+        expect(count_pins(feedback)).to_eq [4, 0]
       end
-      xit 'when ?' do; end
     end
   end
 
-  # am I just testing random cases at this point? is this ok?
   describe '#compare_to_guess' do
+  # am I just testing random cases at this point? is this ok?
```

Comment:
Maybe. Maybe hone down the test cases the test the basic core functionality. Are some of them basically testing the same arithmetic and function multiple times?